web/public/locales/en/views/faceLibrary.json

```diff
@@ -38,14 +43,19 @@
     "success": {
       "uploadedImage": "Successfully uploaded image.",
       "addFaceLibrary": "{{name}} has successfully been added to the Face Library!",
-      "deletedFace": "Successfully deleted face.",
+      "deletedFace_one": "Successfully deleted {{count}} face.",
+      "deletedFace_other": "Successfully deleted {{count}} faces.",
+      "deletedName_zero": "Empty name deleted successfully.",
```

Comment:
Can we call this library instead of name?

Reply:
Yeah, wasn't sure we wanted to call individual names a "library" since we call the entire page "Face Library"